css/extra-styles.css

```diff
@@ -8,14 +8,20 @@
 /* We start with the spatial history assignment */
 
 /* add any styling for the main div here */
+img#main {
```

Comment:
unlikely to apply to anything, since we havea `div#main` elment, but not an `img#main`!